cgi-bin/DW/Controller/Support/Faq.pm

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Support::Faq
+#
+# This controller is for the Support FAQ page.
+#
+# Authors:
+#      hotlevel4 <hotlevel4@hotmail.com>
+#
+# Copyright (c) 2015 by Dreamwidth Studios, LLC.
+#
+# This is based on code originally implemented on LiveJournal.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Support::Faq;
+
+use strict;
+use warnings;
+use vars qw(%GET);
```

Comment:
I didn't realize this would work!

I'd encourage doing it this way though:

    $r = DW::Request->get;
    my $get = $r->get_args;
    ... = $get->{...}